kf_release_maker/release_maker.py

```diff
@@ -0,0 +1,276 @@
+import os
+from urllib.parse import urlencode
+from pprint import pformat
+
+import pandas
```

Comment:
Do we really need pandas here?
It's quite a heavy dependency for a small script.

Reply:
Maybe not, just thought it was easy to use for what I wanted